rep-0149.rst

```diff
@@ -608,6 +608,8 @@ Attributes
   * variable names which start with a `$` sign and are followed by
     alphanumerics and underscores
   * literals which can only contain alphanumerics, underscores and dashes
+  * quoted literals (single or double quotes) which can contain any characters
+    except the used quote character
```

Comment:
The only open question to me is: what if you want to have quotes inside of your conditional expression?  Is there an escape (`\` or such) that allows that?

Reply:
No, there is on escaping.

A single quoted string can't contain a single quote and a double quoted string can't contain a double quote. Obviously a single quoted string can contain a double quote or a double quoted string can contain a single quote.

Reply:
Since we aren't allowing `\` right now, that's always an extension we can add later on.

Reply:
> Since we aren't allowing `\` right now

We do allow quoted literals to contain backslashes but they have to special semantic: "quoted literals which can contain any characters"